client/state/editor/actions.js

```diff
@@ -156,9 +156,10 @@ export function saveConfirmationSidebarPreference( siteId, isEnabled = true ) {
 	};
 }
 
-export const setEditorIframeLoaded = ( isIframeLoaded = true ) => ( {
+export const setEditorIframeLoaded = ( isIframeLoaded = true, iframePort = null ) => ( {
```

Comment:
`setEditorIframeLoaded` is also used by the Calypso classic editor. Is that something we should account for?

Reply:
afaics my changes should be compatible with those calls in `client/post-editor/post-editor.jsx` as we don't do anything in case the `iframePort` hasn't been set